.github/workflows/publish.yml

```diff
@@ -25,24 +26,46 @@ jobs:
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
       - name: Build extension
         run: sh build-ext.sh
+
+      # Chromium
       - uses: actions/upload-artifact@v2
-        name: Upload Extension Zip
+        name: Upload Artifact - Chromium Extension Zip
         with:
           name: stacks-wallet-chromium
           path: stacks-wallet-chromium.zip
+      - name: Upload Chromium extension to production
+        uses: google-github-actions/upload-cloud-storage@864317d33c42de84de94313c5f834802365977b0
+        with:
+          credentials: ${{ secrets.GCS_BUCKET_CREDENTIALS }}
+          path: ./stacks-wallet-chromium.zip
+          destination: wallet-extensions
+
+      # Firefox
       - name: Sign Production Firefox version
         continue-on-error: true
-        run: yarn web-ext sign --channel=listed
+        run: yarn web-ext sign --channel=unlisted
         env:
           WEB_EXT_API_KEY: ${{ secrets.WEB_EXT_API_KEY }}
           WEB_EXT_API_SECRET: ${{ secrets.WEB_EXT_API_SECRET }}
       - uses: actions/upload-artifact@v2
-        name: Upload Firefox Add-On XPI
+        name: Upload Artifact - Firefox Add-On XPI
         with:
           name: connect-addon
           path: web-ext-artifacts/*.xpi
+      - name: Get Firefox addon filename
+        id: addon-file
+        run: echo "::set-output name=addon_path::$(find ./web-ext-artifacts -type f -iname "*.xpi" | tail -n1)"
+      - name: Rename Firefox addon file
+        run: cp ${{ steps.addon-file.outputs.addon_path }} stacks-wallet-firefox.xpi
```

Comment:
If this file ever gets downloaded directly, verision number would be helpful

Reply:
The problem is that this is the file uploaded to GCP and so we need it to be a non-changing file name, so we can point people to `https://upload-url.com/stacks-wallet-firefox.xpi` and it's always the latest.

What if we added the version number to the GA archive file, would that work?

Reply:
GA? 🤔  But yeah if that adds the version, sure. Not a biggie, just useful information and mitigates some `stacks-wallet-firefox (1).xpi` issues

Reply:
GA = github actions, sorry! But yeah, we can't change the filename uploaded to GCP, because it's a simple file store that gives results based on the filename. We could append a version name, but then we'd need a server-side component that can respond to something like `https://wallet-download.hiro.so/web/latest`.